pyproject.toml

```diff
@@ -36,6 +36,7 @@ django-debug-toolbar = "^3.2.1"
 django-pivot = "^1.8.1"
 matplotlib = "^3.3.2"
 nautobot-plugin-nornir = ">=0.9.4"
+nornir-nautobot = "^2.2.0"
```

Comment:
this should be set in nautobot-plugin-nornir, which has this already. rebase from develop and should be good.

Reply:
I will take a look